src/components/AnimePage.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { FaArrowRightLong } from 'react-icons/fa6';
+import { useDispatch, useSelector } from 'react-redux';
+import { Spinner } from '@material-tailwind/react';
+import categories from './categoriesData';
+import { fetchAnimeListByCategory, fetchAnimeDetailsById } from '../redux/HomePage.Slice';
+
+const AnimeCard = () => {
+  const dispatch = useDispatch();
+  const animeList = useSelector((store) => store.anime.animeList);
+  const isLoading = useSelector((store) => store.anime.loading);
+
+  const [selectedCategory, setSelectedCategory] = useState('Action');
+  const [searchTerm, setSearchTerm] = useState('');
+
+  useEffect(() => {
+    dispatch(fetchAnimeListByCategory(selectedCategory));
+  }, [dispatch, selectedCategory]);
+
+  const filteredAnimeList = animeList.filter(
+    (anime) => anime.canonicalTitle.toLowerCase().includes(searchTerm.toLowerCase()),
+  );
+
+  if (isLoading) {
+    return (
+      <div className="flex items-center justify-center w-screen h-screen">
+        <Spinner className="h-12 w-12 text-sky-700" />
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <div className="bg-gradient-to-b from-sky-500 to-sky-700 p-4 grid grid-cols-3 md:grid-cols-6 gap-2 md:gap-6 text-white">
+        <select
+          value={selectedCategory}
+          onChange={(e) => setSelectedCategory(e.target.value)}
+          className="p-1 border-2 border-white ease-in rounded-md text-sky-700"
+        >
+          {categories.map((category) => (
+            <option value={category} key={category}>
+              {category}
+            </option>
+          ))}
+        </select>
+        <input
+          type="text"
+          placeholder="Search anime by name"
+          value={searchTerm}
+          onChange={(e) => setSearchTerm(e.target.value)}
+          style={{ width: '300px' }}
+          className="w-100 p-1 border-2 border-white ease-in rounded-md text-sky-700"
```

Comment:
- [ ] Hello, I would advise you to work on the design again because the requirements says:
 you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:
Typography: font face, size, and weight
 Layout: composition and space between elements.

- See your project; **Titles of the card must be on the card, and not to be isolated outside the card**

![Capture](https://github.com/axcell-0/react-capstone/assets/63972670/8a18941a-ec77-42a3-ba18-1994e5cda45e)

- Original design
![1](https://github.com/axcell-0/react-capstone/assets/63972670/6ca7cc79-5d24-4ab4-b31e-7662fe408aaa)